package.json

```diff
@@ -199,6 +199,7 @@
     }
   },
   "dependencies": {
+    "@astronautlabs/jsonpath": "^1.1.0",
```

Comment:
What the differences with https://www.npmjs.com/package/jsonpath ? _(it has more weekly downloads)_

Reply:
That package is unmaintained. https://github.com/dchester/jsonpath/issues/139